hibernate-envers/src/test/java/org/hibernate/envers/test/integration/collection/DetachedMultipleCollectionChangeTest.java

```diff
@@ -227,6 +226,7 @@ public void testRevisionsCounts() throws Exception {
 	}
 
 	@Test
+	@SkipForDialect(value = CockroachDB192Dialect.class, comment = "requires serial_normalization=sql_sequence setting")
```

Comment:
it seems this should be
```
@SkipForDialects( value = {
			@SkipForDialect(value = CockroachDB192Dialect.class, comment = "requires serial_normalization=sql_sequence setting"),
			@SkipForDialect(value = Oracle8iDialect.class,
					comment = "Oracle does not support identity key generation")
	})
```
otherwise the 
```
@SkipForDialect(value = Oracle8iDialect.class,
				comment = "Oracle does not support identity key generation")
```
of the test class declaration is ignored and the test fails on Oracle.

Reply:
done, didn't realize the method-level and class-level annotations could conflict.